### ISSUE TITLE
Installation instructions for minima 3

### ISSUE THREAD

**Issue by DoodlesEpic** [jekyll/minima#661]
Right now the README documentation is very weird because although most people are probably using Minima 2, the documentation explains mostly how to use Minima 3, but it also only shows how to install the Minima 2 theme using the gem 🤔. It would be nice to have installation instructions for both Minima 2 and 3, an explanation between the (small, albeit relevant) differences between both differences including the distribution method (Minima 3 is only available through GitHub AFAIK).

### TLDR

Add to the README this line:

```yaml
gem "minima", github: "jekyll/minima"
```

I'll fork minima this week and create a PR for these changes if no one does it before that.

**Comment by DoodlesEpic** [comment]
Also, it would be cool to have the gem updated to v3, but I guess this isn't happening lol.

**Comment by chrisxfire** [comment]
@DoodlesEpic what would it take for the gem to be updated to v3?

**Comment by jzohrab** [comment]
Unfortunately the suggested `gem "minima", github: "jekyll/minima"` doesn't appear to work, at least not in my particular case.  After relaxing various gemfile constraints and trying `bundle install` I get a tangle of dependency issues:

```
Resolving dependencies...
Could not find compatible versions

    Because jekyll >= 3.9.0, < 4.0.0.pre.alpha1 depends on kramdown >= 1.17, < 3
      and jekyll >= 3.6.0.pre.beta1, < 3.9.0 depends on kramdown ~> 1.14,
      jekyll >= 3.6.0.pre.beta1, < 4.0.0.pre.alpha1 requires kramdown >= 1.14, < 3.
    And because jekyll >= 4.0.0.pre.alpha1, < 4.2.0 depends on kramdown ~> 2.1
      and jekyll >= 2.0.0, < 3.6.0.pre.beta1 depends on kramdown ~> 1.3,
      jekyll >= 2.0.0, < 4.2.0 requires kramdown >= 1.3, < 3.
(1) Because github-pages >= 9, < 14 depends on kramdown = 1.2.0
      and github-pages < 9 depends on kramdown = 1.0.2,
      github-pages < 14 requires kramdown = 1.0.2 OR = 1.2.0.
    Thus, jekyll >= 2.0.0, < 4.2.0 is incompatible with github-pages < 14.
(2) So, because github-pages >= 203 depends on minima = 2.5.1,
      jekyll >= 2.0.0, < 4.2.0 requires minima = 2.5.1.

    Because github-pages >= 135, < 178 depends on minima = 2.1.1
      and github-pages >= 44, < 147 depends on liquid = 3.0.6,
      github-pages >= 44, < 178 requires minima = 2.1.1 or liquid = 3.0.6.
(3) So, because github-pages >= 28, < 44 depends on jekyll = 2.4.0,
      github-pages >= 28, < 178 requires minima = 2.1.1 or liquid = 3.0.6 or jekyll = 2.4.0.

(4) Because github-pages >= 14, < 32 depends on kramdown = 1.3.1
      and github-pages < 14 requires kramdown = 1.0.2 OR = 1.2.0 (1),
      github-pages < 32 requires kramdown = 1.0.2 OR = 1.2.0 OR = 1.3.1.
    And because github-pages >= 28, < 178 requires minima = 2.1.1 or liquid = 3.0.6 or jekyll = 2.4.0 (3),
      github-pages < 178 requires minima = 2.1.1 or liquid = 3.0.6 or jekyll = 2.4.0 or kramdown = 1.0.2 OR = 1.2.0 OR = 1.3.1.
    And because github-pages >= 178 depends on jekyll-sass-converter = 1.5.2
      and jekyll >= 4.3.0 depends on jekyll-sass-converter >= 2.0, < 4.0,
      jekyll >= 4.3.0 requires minima = 2.1.1 or liquid = 3.0.6 or kramdown = 1.0.2 OR = 1.2.0 OR = 1.3.1.
    And because jekyll >= 4.3.0 depends on kramdown >= 2.3.1, < 3.A
      and jekyll >= 3.5.0 depends on liquid ~> 4.0,
      jekyll >= 4.3.0 requires minima = 2.1.1.
    And because jekyll >= 2.0.0, < 4.2.0 requires minima = 2.5.1 (2),
      jekyll >= 2.0.0, < 4.2.0 OR >= 4.3.0 requires minima = 2.1.1 OR = 2.5.1.
(5) So, because minima = 2.1.1 OR = 2.5.1 could not be found in https://github.com/jekyll/minima.git (at master@8537486)
      and every version of minima depends on jekyll >= 3.5, < 5.0,
      every version of minima requires jekyll >= 4.2.0, < 4.3.0.

    Because github-pages >= 29 depends on terminal-table ~> 1.4
      and github-pages < 32 requires kramdown = 1.0.2 OR = 1.2.0 OR = 1.3.1 (4),
      either kramdown = 1.0.2 OR = 1.2.0 OR = 1.3.1 or terminal-table ~> 1.4.
    And because jekyll >= 4.2.0, < 4.3.0 depends on kramdown ~> 2.3
      and jekyll >= 4.2.0, < 4.3.0 depends on terminal-table ~> 2.0,
      jekyll >= 4.2.0, < 4.3.0 cannot be used.
    And because every version of minima requires jekyll >= 4.2.0, < 4.3.0 (5),
      minima cannot be used.
    So, because Gemfile depends on minima >= 0,
      version solving has failed.
```

note the snippet

```
    And because jekyll >= 2.0.0, < 4.2.0 requires minima = 2.5.1 (2),
      jekyll >= 2.0.0, < 4.2.0 OR >= 4.3.0 requires minima = 2.1.1 OR = 2.5.1.
```

**Comment by ethan-dowler** [comment]
I ran into the same problem
![image](https://github.com/jekyll/minima/assets/32302647/13685d88-dca7-47b0-b544-513735c2abc2)

**Comment by pjschinske** [comment]
I am also having the problem that @jzohrab and @ethan-dowler had. Has anyone found a solution?

**Comment by ashmaroli** [comment]
The recommended solution for those who are using Jekyll 3.x is to connect with the repository via the `jekyll-remote-theme` plugin.

```yaml
# _config.yml


plugins:
  # existing plugins
  - xyz
  # add remote-theme plugin
  - jekyll-remote-theme

# Use remote-theme plugin to load `minima` repository. But first, ensure the `theme` key is disabled.
# theme: minima
remote_theme: "jekyll/minima"
```
> [! NOTE]
> If you're not using the `github-pages` gem, then you will have to explicitly list the `jekyll-remote-theme` gem in your Gemfile.*

> [! WARNING]
> `jekyll/minima:master` is under active development and considered unstable. Future changes may break existing builds.
> To ensure stable builds, use a commit SHA to configure `remote_theme` key.
> e.g. `remote_theme: jekyll/minima@1234567`

**Comment by pjschinske** [comment]
🤦 I was setting `remote-theme` instead of `remote_theme`. There goes a few hours of my life I won't get back...

**Comment by Polovinkin** [comment]
I just spent quite some time trying to configure Minima to use v3 on GitHub Pages, and all the time when i was using remote theme in order for it to work it was breaking all my formatting, couldn't understand why. 

Just to share for people who stumble upon this thread:
- I didn't test/use anything locally, only on GitHub Pages, so I didn't create any Gemfile at all, you don't need it in this case.
- I used `remote_theme: jekyll/minima` instead of `remote_theme: "jekyll/minima"`, which was mentioned in this thread before. Don't know if it even makes a difference, but all references to this variable online wrote it without brackets.
- And finally what fixed this all - **check that you have `layout: home` in your index page**, instead of "default" which was there when it was created, at least for me. As soon as I've made it like that- styling and themes started to work!
